pyinfraformat/core/io.py

```diff
@@ -162,7 +189,11 @@ def parse_line(line):
             startindex=startindex,
             outputFormat="GEOJSON",
         )
-        data = wfs_io.read().decode("utf-8")
+        data = wfs_io.read()
```

Comment:
Would we want to add option to handle these cases too?

Reply:
Or should we assume gtk knows what it is doing

Reply:
Tested data chardet, utf-8 or unknown/corrupted encoding